pkg/core/blockchain.go

```diff
@@ -942,12 +942,13 @@ func (bc *Blockchain) persist() error {
 	return nil
 }
 
-// GetTransaction returns a TX and its height by the given hash.
-func (bc *Blockchain) GetTransaction(hash util.Uint256) (*transaction.Transaction, uint32, error) {
+// GetTransaction returns a TX and its height by the given hash. The height is -1 if tx is in the mempool.
+func (bc *Blockchain) GetTransaction(hash util.Uint256) (*transaction.Transaction, int32, error) {
```

Comment:
We have `uint32` for block numbers and it's very convenient to have it here also.